silkworm/silkrpc/core/fee_history_oracle.cpp

```diff
@@ -119,7 +119,7 @@ Task<FeeHistory> FeeHistoryOracle::fee_history(uint64_t newest_block, uint64_t b
     co_return fee_history;
 }
 
-Task<BlockRange> FeeHistoryOracle::resolve_block_range(uint64_t last_block, uint64_t block_count, uint64_t max_history) {
+Task<BlockRange> FeeHistoryOracle::resolve_block_range(BlockNum last_block, uint64_t block_count, uint64_t max_history) {
```

Comment:
The same change should be applied also in `fee_history_oracle.hpp`

Reply:
ok